esp/public/media/default_styles/onsite_ajax_status.css

```diff
@@ -2,6 +2,52 @@ body {
     font-family: Arial, Helvetica, sans-serif;
 }
 
+#side_area {
+    position: fixed;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 240px;
+    background-color: #FFFFFF;
+    z-index: 20;
+    overflow: auto;
+}
+
+#main_area {
+    position: relative;
+    left: 240px;
+}
+
+#hide_sidebar {
+    display: block;
+    width: 100%;
+}
+
+#show_sidebar {
+    display: none;
+}
+
+#side_area.sidebar_hidden {
+    width: 40px;
+}
+
+#main_area.sidebar_hidden {
+    left: 40px;
+}
+
+#side_area.sidebar_hidden table {
+    display: none;
+}
+
+#side_area.sidebar_hidden #hide_sidebar {
```

Comment:
These selectors are longer than necessary, given that the last selector is an id.